src/app.jsx

```diff
@@ -794,29 +794,32 @@ class Application extends React.Component {
             </AlertGroup>
         );
 
+        console.log(this.state.version, WithVersion);
         return (
             <WithDialogs>
-                <Page id="overview" key="overview">
-                    {notificationList}
-                    <PageSection className="content-filter" padding={{ default: 'noPadding' }}
-                                 variant={PageSectionVariants.light}>
-                        <ContainerHeader
-                            handleFilterChanged={this.onFilterChanged}
-                            handleOwnerChanged={(_event, value) => this.onOwnerChanged(value)}
-                            ownerFilter={this.state.ownerFilter}
-                            textFilter={this.state.textFilter}
-                            twoOwners={this.state.systemServiceAvailable && this.state.userServiceAvailable}
-                            user={this.state.currentUser}
-                        />
-                    </PageSection>
-                    <PageSection className='ct-pagesection-mobile'>
-                        <Stack hasGutter>
-                            { this.state.showStartService ? startService : null }
-                            {imageList}
-                            {containerList}
-                        </Stack>
-                    </PageSection>
-                </Page>
+                <WithVersion value={this.state.version}>
+                    <Page id="overview" key="overview">
+                        {notificationList}
+                        <PageSection className="content-filter" padding={{ default: 'noPadding' }}
+                                     variant={PageSectionVariants.light}>
+                            <ContainerHeader
+                                handleFilterChanged={this.onFilterChanged}
+                                handleOwnerChanged={(_event, value) => this.onOwnerChanged(value)}
+                                ownerFilter={this.state.ownerFilter}
+                                textFilter={this.state.textFilter}
+                                twoOwners={this.state.systemServiceAvailable && this.state.userServiceAvailable}
+                                user={this.state.currentUser}
+                            />
+                        </PageSection>
+                        <PageSection className='ct-pagesection-mobile'>
+                            <Stack hasGutter>
+                                { this.state.showStartService ? startService : null }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1249-20230619-215509-db4ac5a7-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)